icu4c/source/i18n/number_longnames.cpp

```diff
@@ -232,15 +282,96 @@ UnicodeString getPerUnitFormat(const Locale& locale, const UNumberUnitWidth &wid
     return UnicodeString(ptr, len);
 }
 
+// Load data originating from a deriveComponent CLDR element. The feature and
+// structure parameters must be null-terminated.
```

Comment:
Use function/Documentation  comment, /** */

Reply:
Going with the "///" style, also known by doxygen. (But I suspect doxygen won't be generating docs for this function.)